pyhive/sqlalchemy_hive.py

```diff
@@ -247,10 +258,15 @@ class HiveDialect(default.DefaultDialect):
     supports_multivalues_insert = True
     type_compiler = HiveTypeCompiler
     supports_sane_rowcount = False
+    supports_statement_cache = False
 
     @classmethod
     def dbapi(cls):
         return hive
+    
+    @classmethod
+    def import_dbapi(cls):
```

Comment:
The Dialect.import_dbapi() class method is renamed from the previous method .Dialect.dbapi()

https://docs.sqlalchemy.org/en/20/core/internals.html#sqlalchemy.engine.Dialect.import_dbapi